docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,197 @@
+will still happen and lower priority pods will be removed despite their PDBs 
+being violated.
+
+#### Inter-Pod Affinity on Lower Priority Pods
+The current implementation of preemption considers a node for preemption only when
+the answer to this question is positive: "If all the pods with lower priority than
+the pending pod are removed from the node, can the pending pod be scheduled on
+the node?"
+(Note that preemption does not always remove all lower-priority pods, e.g. if the 
+pending pod can be scheduled by removing fewer than all lower-priority pods, but this
+test must always pass for preemption to be considered on a node.)
+
+If the answer is no, that node will not be considered for preemption. If the pending
+pod has inter-pod affinity to one or more of those lower priority pods on the node, the
+inter-pod affinity rule cannot be satisfied in the absence of the lower priority
+pods and scheduler will find the pending pod infeasible on the node. As a result,
+it will not try to preempt any pods on that node.
+Scheduler will try to find other nodes for preemption and could possibly find another
+one, but there is no guarantee that such a node will be found.
+
+We may address this issue in future versions, but we don't have a clear plan and cannot
+promise that it will be fixed in Beta or GA. Part
+of the reason is that finding the set of lower priority pods that satisfy all
+inter-pod affinity/anti-affinity rules is computationally expensive and adds
+substantial complexity to the preemption logic. Besides, even if preemption keeps the lower
+priority pods to satisfy inter-pod affinity, the lower priority pods may be preempted
+later by other pods, which removes the benefits of having the complex logic of 
+respecting inter-pod affinity to lower priority pods.
+
+Our recommended solution for this problem is to create inter-pod affinity towards
+equal or higher priority pods.
+
+#### Cross Node Preemption
+When considering a node N for preemption in order to schedule a pending pod P,
+P may become feasible on N only if pods on other nodes are preempted. For example, P may
+have zone anti-affinity with some currently-running, lower-priority pod Q. P may not be
+scheduled on Q's node even if it preempts Q, for example if P is larger than Q so
+preempting Q does not free up enough space on Q's node and P is not high-priority enough
+to preempt other pods on Q's node. But P might theoretically be able to schedule on 
+another node M by preempting Q and some pod(s) on M (preempting Q removes the
+anti-affinity violation, and preempting pod(s) on M frees up space for P to schedule
+there). The current preemption algorithm does not detect and execute such preemptions;
+that is, when determining whether P can schedule onto N, it only considers preempting
+pods on N.
+
+We may consider adding cross node preemption in future versions if we find an
+algorithm with reasonable performance, but we cannot promise anything at this point.
```

Comment:
I think @wojtek-t would feel better if you add at the end something like "(it will not be considered a blocker for Beta and GA)."

Reply:
Done.